doc/reference/storage_drivers.md

```diff
@@ -55,12 +55,22 @@ Therefore, the first instance takes longer to create than subsequent ones.
 ### Optimized instance transfer
```

Comment:
Can we change this heading to something like "Optimized storage transfer" so it covers instances and custom volumes.

Reply:
Yes, that makes a lot of sense. Let's call it optimized volume transfer (since that's what we already use in the section).